src/views/datalist/data-list-directive.js

```diff
@@ -214,12 +214,22 @@ angular.module('patternfly.views').directive('pfDataList', [
           };
 
           $scope.config = $.extend(true, angular.copy($scope.defaultConfig), $scope.config);
+          if ($scope.config.selectItems && $scope.config.showSelectBox) {
+            throw new Error('pfDataList - ' +
+            'Illegal use of pfDataList directive! ' +
+            'Cannot allow both select box and click selection in the same data list.');
+          }
         }
       ],
 
       link: function (scope, element, attrs) {
         attrs.$observe('config', function () {
           scope.config = $.extend(true, angular.copy(scope.defaultConfig), scope.config);
+          if (scope.config.selectItems && scope.config.showSelectBox) {
+            throw new Error('pfDataList - ' +
+            'Illegal use of pfDataList directive! ' +
+            'Cannot allow both select box and click selection in the same data list.');
```

Comment:
Actually, I need to remove this second occurrence. There is no way to catch the error so it's not really helping to have it thrown.